### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by mimse**
Moved FormGroupDirective to shell

### CODE REVIEW

File: libs/eo/core/shell/src/lib/eo-core-shell.providers.ts [hunk 1]
```diff
@@ -25,6 +26,7 @@ import { danishDatetimeProviders } from '@energinet-datahub/watt/danish-date-tim
 import { WattModalService } from '@energinet-datahub/watt/modal';
 
 export const eoCoreShellProviders = [
+  FormGroupDirective,
```
Comment (frontend-specialisten) [inline libs/eo/core/shell/src/lib/eo-core-shell.providers.ts:29]: Question: Why is this needed?

Comment (frontend-specialisten) [inline libs/eo/core/shell/src/lib/eo-core-shell.providers.ts:29]: Question: This doesn't seem to be necessary. Do you have a specific reason for adding this? Otherwise, please remove it :)

Comment (mimse) [reply libs/eo/core/shell/src/lib/eo-core-shell.providers.ts:29]: I needed to add this for reactive form validation in watt-field to work
https://github.com/Energinet-DataHub/greenforce-frontend/blob/ae88fffd8c922243373df5b7a7bb742ec0bcd907/libs/watt/src/lib/components/field/watt-field.component.ts#L63

Comment (frontend-specialisten) [reply libs/eo/core/shell/src/lib/eo-core-shell.providers.ts:29]: But if I remove it, it still work? (in EO)

Comment (mimse) [reply libs/eo/core/shell/src/lib/eo-core-shell.providers.ts:29]: Aha, must be because you added the optional param

**Review by frontend-specialisten — APPROVED** [review]
